kong/plugins/acme/client.lua

```diff
@@ -332,9 +373,21 @@ local function renew_certificate_storage(conf)
       kong.log.err("can't read renew conf: ", err)
       goto renew_continue
     end
+    if not renew_conf then
```

Comment:
why is this needed?

Reply:
if one worker is slightly behind the other, `renew_conf` can be null for the second worker if the first worker deletes it.